packages/auth/src/accessToken/routes.test.ts

```diff
@@ -25,7 +25,7 @@ describe('Access Token Routes', (): void => {
   let accessTokenRoutes: AccessTokenRoutes
 
   beforeAll(async (): Promise<void> => {
-    deps = await initIocContainer(Config)
+    deps = initIocContainer(Config)
```

Comment:
:eyes: are we pointlessly `await`ing this _everywhere_?

Reply:
yes, but doesn't really impact anything